include/deal.II/base/numbers.h

```diff
@@ -542,6 +565,51 @@ namespace internal
   };
 }
 
+namespace numbers
+{
+
+#ifdef DEAL_II_ADOLC_WITH_ADVANCED_BRANCHING
+  // Defined in differentiation/ad/adolc_number_types.cc
+  bool
+  are_equal_valued (const adouble &value_1,
+                    const adouble &value_2);
```

Comment:
Do these functions show up in the doxygen index? If so, you'll want to document them.